spec/chewy/query_spec.rb

```diff
@@ -454,7 +456,7 @@
   end
 
   describe '#exists?' do
-    let(:data) { 10.times.map { |i| {id: i.next.to_s, name: "Name#{i.next}", age: 10 * i.next}.stringify_keys! } }
+    let(:data) { Array.new(10) { |i| {id: i.next.to_s, name: "Name#{i.next}", age: 10 * i.next}.stringify_keys! } }
```

Comment:
True.

On Friday, 27 May 2016, Arkadiy Zabazhanov <notifications@github.com> wrote:

> In spec/chewy/query_spec.rb
> <https://github.com/toptal/chewy/pull/393#discussion_r64883611>:
>
> > @@ -454,7 +456,7 @@
> >    end
> >
> >    describe '#exists?' do
> > -    let(:data) { 10.times.map { |i| {id: i.next.to_s, name: "Name#{i.next}", age: 10 * i.next}.stringify_keys! } }
> > +    let(:data) { Array.new(10) { |i| {id: i.next.to_s, name: "Name#{i.next}", age: 10 * i.next}.stringify_keys! } }
>
> Oh, really? Don't think it is significant in this particular case,
> 10.times.map looks more like ruby-way as for me, Array.new(10) is c++ or
> java.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/toptal/chewy/pull/393/files/8a9a6bbeb01054ef878640b595be7e7da2298f8c#r64883611>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAGVynrZ3G_eFfzWT94Z0GGjfaG-HtWHks5qFsEDgaJpZM4IoK68>
> .
>


-- 
Cheers,
Bozhidar

Bozhidar Batsov, VP of Engineering || Toptal <https://toptal.com> ||
bozhidar_ (skype)